src/test/resources/UserRegistration.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User Registration
+  As a user (/admin)
+  I want to be able to register (users) with username and password
+
+  Background:
+    Given database is empty
+    And user with id 1234 exists and has username "user", password "secure_password" and refreshToken "token"
+
+  Scenario: Successful registration with username, password and password confirmation.
+    When user requests registration with username "kangaroo", password "pig-system" and password confirmation "pig-system"
```

Comment:
The requester has to provide a valid accessToken to verify that he is a admin. use `accessToken with value {string} exists for user {long}`. Also specify that the user "user" is indeed an admin like `user with id belongs to group with the id`